packages/apollo-server-express/src/__tests__/datasource.test.ts

```diff
@@ -106,14 +106,14 @@ describe('apollo-server-express', () => {
     const apolloFetch = createApolloFetch({ uri });
     const firstResult = await apolloFetch({ query: '{ id }' });
 
-    expect(firstResult.data).toEqual({ id: 'hi' });
     expect(firstResult.errors).toBeUndefined();
```

Comment:
When you assert errors to be undefined before you check if data is correct, Jest will actually tell you what the errors are instead of just saying the data is null